initdb.py

```diff
@@ -134,7 +138,8 @@ def main(username='', password=''):
         mysql_connection = mysql.connector.connect(user=username,
                                                    password=password)
     except mysql.connector.Error as err:
-        raise Exception('There was a problem connecting to the server:\n{}'.format(err))
+        raise InputError(message='There was a problem connecting to the server:\n{}'.format(err),
```

Comment:
Don't put `err` in the message. The end-user doesn't need to see the stacktrace or original error. That's why we have `args`.